Dockerfile

```diff
@@ -100,4 +102,5 @@ USER 1001
 
 ENV LD_LIBRARY_PATH=/open-vds:$LD_LIBRARY_PATH
 ENV OPENVDS_AZURESDKFORCPP=1
+ENV LD_PRELOAD=/usr/lib/libjemalloc.so
```

Comment:
Is this file independent of architecture? In other words, is it compiled or does it belongs to the OS?

Reply:
Ok, thanks for the explanation.